Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md

```diff
@@ -0,0 +1,9 @@
+
+#### Scripts
+##### MS365DefenderUserListToTable
+- The MS365DefenderUserListToTable folder is now inside the Scripts folder.
+##### MS365DefenderCountIncidentCategories
+- The MS365DefenderCountIncidentCategories folder is now inside the Scripts folder.
+
+#### integration's README.md
+- Added clarification regarding the required permissions.
```

Comment:
I don't think this requires a note - did you add this manually?

Reply:
Removed